tests/integration_tests/test_experiment.py

```diff
@@ -613,16 +612,16 @@ def test_experiment_date(csv_filename):
         input_features[0]['encoder'] = encoder
         run_experiment(input_features, output_features, data_csv=rel_path)
 
-
-def test_experiment_h3(csv_filename):
+@pytest.mark.parametrize('encoder', ['embed', 'weighted_sum', 'rnn'])
```

Comment:
Maybe here we can use `H3InputFeature.encoder_registry.keys()` instad, so that is we add a new one we are sure it is tested.

Reply:
changed to use `H3InputFeature.encoder_registry.keys()`